tests/tracking/test_rest_tracking.py

```diff
@@ -76,6 +76,7 @@ def create_experiments(client, names):
 
 
 def test_create_get_search_experiment(mlflow_client):
+    time.sleep(0.001)
```

Comment:
Can we add inline comments explaining why we're sleeping?

Reply:
Added!